src/openrct2/peep/Guest.cpp

```diff
@@ -2595,24 +2595,13 @@ bool Guest::FindVehicleToEnter(Ride* ride, std::vector<uint8_t>& car_array)
         vehicle = GET_VEHICLE(vehicle_id);
 
         uint8_t num_seats = vehicle->num_seats;
-        if (vehicle_is_used_in_pairs(vehicle))
-        {
-            num_seats &= VEHICLE_SEAT_NUM_MASK;
-            if (vehicle->next_free_seat & 1)
-            {
-                current_car = i;
```

Comment:
You can't remove this bit of code without breaking all other rides (non ferris wheels).
From tests it looks like you could take this case as a success and push_back into the car_array if it gets here instead of fast forwarding to the entrance. That way all of the money checks will still take place.

Reply:
Good point! Thanks for the attention. I think your suggestion is probably the right choice there. I'll make some more testing and get back with results. :)

Reply:
Seems that it works better now. I'm still doing more tests with ferris wheel and non ferris wheels rides to be sure.